tests/e2e/util.go

```diff
@@ -335,6 +335,33 @@ func createStatefulSetWithOneReplica(client clientset.Interface, manifestPath st
 	return statefulSet, service
 }
 
+// updateDeploymentReplicawithWait helps to update the replica for a deployment with wait
+func updateDeploymentReplicawithWait(client clientset.Interface, count int32, name string, namespace string) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	var deployment *appsv1.Deployment
+	var err error
+	waitErr := wait.Poll(healthStatusPollInterval, healthStatusPollTimeout, func() (bool, error) {
```

Comment:
I need this poll because after PSOD'ng the host we are trying to get the deployment and sometimes we are getting exceptions due to that tests are failing and to retry it I have added a poll.